### ISSUE TITLE
Documentation for face_recognition_demo is incorrect

### ISSUE THREAD

**Issue by mshabunin** [opencv/open_model_zoo#1102]
Script parameters in the documentation ([doc](https://docs.openvinotoolkit.org/latest/_demos_python_demos_face_recognition_demo_README.html), [src](https://github.com/opencv/open_model_zoo/blame/master/demos/python_demos/face_recognition_demo/README.md#L93-L95)):
```
  -i PATH, --input PATH
                        (optional) Path to the input video ('cam' for the
                        camera, default)
```

Actual script:
https://github.com/opencv/open_model_zoo/blob/7d235755e2d17f6186b11243a169966e4f05385a/demos/python_demos/face_recognition_demo/face_recognition_demo.py#L42-L44

**Comment by vladimir-dudnik** [comment]
cc @AnthonyQuantum @Wovchena

**Comment by AnthonyQuantum** [comment]
Fixed by #1111
